frontend/package.json

```diff
@@ -9,8 +9,8 @@
   "scripts": {
     "serve": "vue-cli-service serve",
     "serveOpen": "vue-cli-service serve --open",
-    "lint": "eslint src/**/*.{js,vue} && stylelint ./src/**/*.{vue,scss} && npm run puglint",
-    "lintfix": "eslint --fix src/**/*.{js,vue} cypress/tests/**/*.js && stylelint --fix ./src/**/*.{vue,scss}",
+    "lint": "eslint src/**/*.{js,vue} cypress/tests/**/*.js cypress/support.js && stylelint ./src/**/*.{vue,scss} && npm run puglint",
```

Comment:
Would it be better to combine `cypress/tests/**/*.js` `cypress/support.js` into `cypress/**/*.js`?

Reply:
That would mean that files under `node_modules` will be linted as well. Also because I referred to 

> The correct expression to lint is frontend/cypress/tests/**/*.js frontend/cypress/support.js, to avoid linting the node_modules folder in frontend/cypress/.

in the [issue description](https://github.com/reposense/RepoSense/issues/1564).

Reply:
Ok